src/Data/Product.agda

```diff
@@ -160,6 +154,16 @@ curry′ = curry
 uncurry′ : (A → B → C) → (A × B → C)
 uncurry′ = uncurry
 
+dmap′ : ∀ {x y} {X : A → Set x} {Y : B → Set y} →
+        ((a : A) → X a) → ((b : B) → Y b) →
+        (ab : A × B) → X (proj₁ ab) × Y (proj₂ ab)
+dmap′ f g = dmap f g
+
+_<*>_ : ∀ {x y} {X : A → Set x} {Y : B → Set y} →
+        ((a : A) → X a) × ((b : B) → Y b) →
+        (ab : A × B) → X (proj₁ ab) × Y (proj₂ ab)
+_<*>_ = uncurry dmap′
+
```

Comment:
Note that you can write `((a , b) : A × B) → X a × Y b` nowadays :D

Reply:
I'm not set up for that yet, but indeed this should be done before merging.

Reply:
Ah, fair enough.